pages/manage/productions/index.vue

```diff
@@ -233,17 +233,15 @@
 </script>
 
 <style lang="scss" scoped>
-  .productions-index-page {
-    .productions-events-container {
-      display: flex;
-      width: 100%;
-      font-weight: 400;
-      margin-bottom: 1rem;
-
-      /deep/ h2 {
-        font-size: 1rem;
-        font-weight: 700;
-      }
+  .productions-events-container {
+    display: flex;
+    width: 100%;
+    font-weight: 400;
+    margin-bottom: 1rem;
+
+    /deep/ h2 {
```

Comment:
Also seems like something we should move to a component?

Reply:
h2 is styled differently over multiple pages, so I don't really see how we can make it a component efficiently

Reply:
Can you give some examples? Maybe we can come up with something that looks mostly the same but is consistent. Or if they're completely different in some cases, maybe those cases should become something else than a h2?
We also don't have to call the component `h2`. It could have a more specific name as we have for h1's (`pub-page-title`).

Reply:
Looking at productions alone, I see an `h2` on the create page that's `1.2rem` while this one is `1rem`. Maybe we should make both `1.2rem` to start with? As an `h2` it feels like it should be bigger than normal text imho?

Reply:
Another `h2` I found, but in the AngularJS app, are the titles on the "Invoeren" form. Those also happen to be `1.2rem` (and `font-weight: 700`). They're also capitalized and have border-bottom and a number in front of them, but those are extras we could add in this specific case without changing the `h2` itself I think.

![Screenshot 2020-09-16 at 09 39 47](https://user-images.githubusercontent.com/959026/93306854-b050df80-f800-11ea-9b6e-6bb8b5d00e75.png)

Reply:
<img width="374" alt="Schermafbeelding 2020-09-16 om 09 34 25" src="https://user-images.githubusercontent.com/66943525/93307027-da0a0680-f800-11ea-9aea-48913240af9c.png">
<img width="228" alt="Schermafbeelding 2020-09-16 om 09 34 39" src="https://user-images.githubusercontent.com/66943525/93307036-dbd3ca00-f800-11ea-93f7-a616312553bc.png">
<img width="230" alt="Schermafbeelding 2020-09-16 om 09 41 15" src="https://user-images.githubusercontent.com/66943525/93307041-dbd3ca00-f800-11ea-9ec6-a18df103519c.png">
<img width="268" alt="Schermafbeelding 2020-09-16 om 09 41 23" src="https://user-images.githubusercontent.com/66943525/93307046-dc6c6080-f800-11ea-91ba-69c8cb32ad82.png">
<img width="218" alt="Schermafbeelding 2020-09-16 om 09 41 29" src="https://user-images.githubusercontent.com/66943525/93307047-dd04f700-f800-11ea-9754-f38c655bbdb0.png">
<img width="198" alt="Schermafbeelding 2020-09-16 om 09 41 35" src="https://user-images.githubusercontent.com/66943525/93307050-dd04f700-f800-11ea-9fe5-57c613c4b7af.png">

Reply:
So maybe all occurrences to `1.2rem` and `font-weight: 700 ` and then border, padding and such as styling in the specific pages? The capitalization we could add as a variant or a prop capitalized?

Reply:
Yep that sounds good to me 👍 
Maybe leave the capitalization out, for now, we can figure that out once we have to rebuild the "Invoeren" page.

Reply:
Sorry I meant that we should probably also move that to a separate component in another PR, not in this one 😄 Because now this whole PR is blocked on a potential discussion how h2's should look and be called. Let's create an issue for that like @geoffreydhuyvetters suggested and leave this out of this PR. My comment was more of a suggestion to do it later anyway but I could've worded that a bit better.